tools/packaging/kernel/build-kernel.sh

```diff
@@ -352,7 +352,7 @@ setup_kernel() {
 	${packaging_scripts_dir}/apply_patches.sh "${patches_dir_for_version}"
 
 	# Apply version specific patches for build_type build
-	if [ "${build_type}" == "true" ] ;then
+	if [ "${build_type}" != "" ] ;then
```

Comment:
This could be simplified slightly to:

```suggestion
	if [ -n "${build_type}" ] ; then
```